src/mugs.js

```diff
@@ -912,7 +912,7 @@ define([
                     if (!form.vellum.opts().features.allow_data_reference_in_setvalue) {
                         var paths = new logic.LogicExpression(mug.p.defaultValue, xpath).getPaths();
                         paths = _.filter(paths, function (path) {
-                            return path.initial_context !== xpathmodels.XPathInitialContextEnum.EXPR;
+                            return path.initial_context === xpathmodels.XPathInitialContextEnum.ROOT;
```

Comment:
`ROOT` is absolute references. (this gets a little nicer once we have `#form` refs)